### PR TITLE
3x replacement factions

### PR DESCRIPTION
**Pull request by joonicks**
replacing autogenerated factions with slightly more fleshed out ones

### CODE REVIEW

**Comment by impaktor** [comment]
It was decided in the dev team, some time ago, that all factions but the 4 first will be removed, (e.g. #3629).

**Comment by Gliese852** [comment]
@impaktor I do not see a collective decision to delete factions in #3629

**Comment by impaktor** [comment]
@Gliese852 Indeed, it's taken for given there, I don't remember who/where the discussion was had, and if anyone is wondering, I don't think I was the driving force behind it, although I agree with it. Spamming the universe with +100 factions, that differ only in "a name" and "a color" (sure, there's the illegal goods list), is not meaningful, compared to a handful of hand scripted, that truly give different "feeling" when being there. Just like in Frontier: being in Imperial vs. Alliance vs. Federation space did feel different, the stations, the adverts, the military, the ships on sale, the market prices, etc.

**Comment by Gliese852** [comment]
@impaktor yes, I understand your position, but I think that space is huge, and there must be a lot of things. And I'm sure there must be room for any number of factions if contributors want to describe them. Personally, I like to explore space and see all these homeworlds of factions, it seems that there is a lot of life.

**Comment by joonicks** [comment]
I just suggested to Gliese in chat a quest to visit all the faction homeworlds.

Maybe its best to gradually replace the autogenerated factions over time and also reduce the number to ~50-80 or so, with a handful of large dominating factions and the rest being smaller less expansive.

If my lore ideas gets added, factions would gain a bunch more substance and become more tangible.

**Comment by zonkmachine** [comment]
@joonicks I could cherry-pick this PR on top of master after removing the conflicting files with `git rm`.
The illegal goods names was changed in c4109e06a182d611a519751949f53d075c3b10cd so the ones in all caps gives a warning.

`Lua Warning: Faction[Julian science community]: Invalid commodity name LIQUOR passed to IllegalGoodsProbability.`

**Comment by joonicks** [comment]
I messed up and based my changes on an old git and now I dont know how to fix it proper

File: data/factions/047_Druello_pirates.lua
Comment (Web-eWorks) [inline data/factions/047_Druello_pirates.lua:22]: Faction names should also use Proper Noun capitalization rules.

File: data/factions/077_Tolan_Kingdom.lua
Comment (Web-eWorks) [inline data/factions/077_Tolan_Kingdom.lua:11]: I'd recommend 'Skymarshal' (or even better, 'Sky Marshal') in this case, as 'skymartial' is a noun-adjective pair being coerced into an adjective-adjective pair and doesn't mean anything.

File: data/factions/077_Tolan_Kingdom.lua
Comment (Web-eWorks) [inline data/factions/077_Tolan_Kingdom.lua:10]: 'Defenceforce' should be two words using Proper Noun capitalization rules (and in my mind using the Western spelling), e.g. 'Defense Force'

**Review by Web-eWorks — COMMENTED** [review]
There are several places where the strings in this PR need some sprucing up; I'll leave it to @impaktor / @nozmajner to determine if the short lore proposed for these factions is compliant with the overall lore of the game.

I'm not opposed to merging this PR as it replaces three otherwise blank factions, but the review comments will need to be addressed before merge.

**Comment by zonkmachine** [comment]
It looks like all changes suggested in review have been done. If the English is acceptable I think this should be merged.

**Review by Web-eWorks — APPROVED** [review]
These look fine to merge to me at this time, if needed we can make corrections/improvements once in master. I'm generally in favor of improving the auto-generated factions, and this is a good step in the right direction.

**Comment by syonfox** [comment]
Hmmmmm just a though what if there were 2 start modes 

year 3224 and 
  Humanity has expanded exponentially with new tech nad has developed the local cluster.
  like in the wars there have been many splinter groups but things ha settled in the last 300 
  years with these advances in tec etc. splinter groups have moved to the edge and the main rivals of centralized powers remains as 3-10 factions around sol.

year 30224 
  as the stalemate and bureaucracy progresses there is a growing dynamic on the edge and 
  more and more people establish factions around the

**Comment by zonkmachine** [comment]
@joonicks Hi! I had a go at rebasing this but I'm not a member and can't push to this branch so I issued a new PR of one commit, in your name, that replaces this one. I hope you're fine with this. ;)
https://github.com/pioneerspacesim/pioneer/pull/5892

**Pull request closed without merging**
